### PR TITLE
Updating migration dependencies in 2.6.x

### PR DESCRIPTION
**Pull request by phlax**
Migrations were failing on sites with real data during the upgrade process from
2.5.x to 2.6.0 due to incorrect migration ordering.

### CODE REVIEW

**Comment by unho** [comment]
lgtm

File: docs/releases/2.6.1.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
+===================================
+Welcome to the Pootle 2.6.1 Interim
+===================================
+
+*Released on 14 September 2015*
+
+The 2.6.1 release is an interim release. It is used to migrate from Pootle
+2.5.0 or newer to Pootle 2.7.0.
```
Comment (unho) [inline docs/releases/2.6.1.rst]: It is also used to migrate to releases newer than 2.7.0.

File: docs/releases/2.6.1.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  If you are upgrading from Pootle older than 2.5.0 you must first upgrade to
+  2.5.1.3 before upgrading to this version.
+
+
+Credits
+=======
+
+This release was made possible by the following people:
+
+Ryan Northey, Leandro Regueiro, Dwayne Bailey
```
Comment (unho) [inline docs/releases/2.6.1.rst]: We only credit people that have contributed to this particular release. Checking the stable/2.6.0 branch I can't see any contribution from me or Dwayne.

File: docs/releases/2.6.1.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+.. warning::
+
+  Do not run a Pootle instance using this version.
+
+
+
+Major Changes
+=============
+
+*Ordering of migration dependencies*
```
Comment (unho) [inline docs/releases/2.6.1.rst]: This line is unnecessary.

File: docs/releases/2.6.1.rst (hunk 2 above)
Comment (phlax) [inline docs/releases/2.6.1.rst]: not in the git tree - but in making the release happen i figured

Comment (unho) [inline docs/releases/2.6.1.rst]: We use a chain of commands to get the contributor list by number of contributions from the actual commits. See last snippet in "Create release notes" section in http://docs.translatehouse.org/projects/pootle/en/stable-2.7.0/developers/releasing.html#create-release-notes

**Comment by unho** [comment]
lgtm

**Pull request merged**